playwright.config.ts

```diff
@@ -21,7 +21,7 @@ dotenv.config()
 export default defineConfig({
   testDir: './e2e',
   /* Timeout to allow portal enough time to compile when running locally */
-  timeout: 5 * 60 * 1000,
+  timeout: 2 * 60 * 1000,
```

Comment:
Reduce overall test timeout, so flaky tests fail faster -- currently, the longest test (`teams.loggedin`) takes about 25s locally, 17s on larger runner, and 35-40s on macOS runner, so 2 minutes should be enough time for now. If we add a longer test in the future, we can configure a [longer timeout](https://playwright.dev/docs/test-timeouts#set-timeout-for-a-single-test) just for that test.

Reply:
I like the idea of having these fail faster if making these longer doesn't substantially make these less flaky. Optimizing for resilience is good, but I wouldn't worry much about shaving seconds or minutes off to save money.